cli/main.go

```diff
@@ -256,3 +245,16 @@ func setupLogging() error {
 	log.Debug("debug logging enabled")
 	return nil
 }
+
+func sanitizeMinBid(minBid float64) (*types.U256Str, error) {
+	if minBid < 0.0 {
+		log.Fatal("Please specify a non-negative minimum bid")
```

Comment:
can return error instead of log.Fatal and leave the error check to the calling function

Reply:
yep